src/app/component/user/components/profile/profile-service/profile.service.ts

```diff
@@ -48,4 +58,13 @@ export class ProfileService {
   public getUserFriends(): Observable<object> {
     return this.http.get<object>(`${this.backEnd}user/${this.userId}/sixUserFriends/`);
   }
+
+  public toggleStatusOfShoppingItem(item): Observable<object[]> {
+    const { status, goalId } = item;
+    const newStatus = status === 'DONE' ? false : true;
+    const params = new HttpParams()
+      .set('goalId', goalId)
+      .set('status', newStatus.toString());
```

Comment:
As it was mentioned, it is better to leave an empty line before the 'return'

Reply:
done